cmd/registry/main.go

```diff
@@ -67,6 +68,15 @@ func main() {
 	handler = panicHandler(handler)
 	handler = gorhandlers.CombinedLoggingHandler(os.Stdout, handler)
 
+	if config.HTTP.Secret == "" {
```

Comment:
Let's move this into app. We'll refactor later and I don't want this to get lost.